Mithril update 2024-05-22 (#398)
IntersectMBO/cardano-updates: blog/2024-05-22-mithril.md

---
title: Mithril Team Update
slug: 2024-05-22-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team continued implementing the certification of Cardano transactions in Mithril networks. They worked on scaling the proof generation for `mainnet` with prototyping some optimizations and benchmarking the improvements on the performances. They also made progress in implementing low latency certification: they completed the retrieval of the tip of the chain and worked on importing transactions from Cardano mini-protocol with Pallas. Additionally, they worked on a new page of the explorer that will display in/out SPOs for the latest Cardano epochs.
> 
> Finally, the team upgraded the `testing-sanchonet` network following the `Sanchonet` network respin, worked on creating a module for building test transactions and started removing the deprecated `snapshot` command from the client CLI.

Reviewer: ```suggestion
This week, the Mithril team continued implementing the certification of Cardano transactions in Mithril networks. They worked on scaling proof generation for `mainnet` by prototyping optimizations and benchmarking performance improvements. They also made progress on low-latency certification by completing the retrieval of the chain tip and importing transactions from the Cardano mini-protocol with Pallas. Additionally, they worked on a new explorer page to display in/out SPOs for the latest Cardano epochs.

Finally, the team upgraded the `testing-sanchonet` network following the `SanchoNet` network respin, created a module for building test transactions, and began removing the deprecated `snapshot` command from the client CLI.
```
